### PR TITLE
Feature/generate dataset

### PR DESCRIPTION
**Pull request by beckstev**

### CODE REVIEW

File: dog_classifier/io/dataset.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+                # (x1, y1): top left corner
+                # (x2, y2): top right corner
+                # (x3, y3): bottom right corner
+                # (x4, y4): bottom left corner
+                corners = sort_corners(corners)
+                # The NN only can input and output one dim objects, therefore,
+                # we have to flatten our two dim array.
+                dog_labels = corners.flatten().tolist()
+
+                # We add the other lael to the bboox coordinates to
```
Comment (FeGeyer) [inline dog_classifier/io/dataset.py:118]: Kleiner Schreibfehler, es müsste label sein :)

File: dog_classifier/io/dataset.py [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+               "width", "height", "path_to_label", "filename"]
+    # Save all the labels to a Dataframe
+    df_label = pd.DataFrame(data=labels, columns=columns, dtype='uint16')
+
+    df_train_val, df_test = train_test_split(df_label, train_size=train_size + val_size,
+                                             test_size=test_size,
+                                             shuffle=True,
+                                             random_state=13)
+
+    # We need to normalize the percentage amount of train and val, because
```
Comment (FeGeyer) [inline dog_classifier/io/dataset.py:137]: Das verstehe ich leider nicht ganz, aber kannst du mir auch so erklären :)

**Review by FeGeyer — APPROVED** [review]
Nur ein kleiner Schreibfehler, ansonsten super. Nur warum normalisiert werden muss, verstehe ich nicht ganz...

File: dog_classifier/io/dataset.py (hunk 2 above)
Comment (beckstev) [reply dog_classifier/io/dataset.py:137]: Mach ich morgen, ist schwierig in Worte zufassen.
